collect_app/src/main/res/values/strings.xml

```diff
@@ -743,4 +743,5 @@
     <string name="preference_not_available">Not available</string>
     <string name="legacy_custom_server_paths">Legacy custom server paths</string>
     <string name="custom_server_paths">Custom server paths</string>
+    <string name="legacy_custom_server_paths_summary">Will be removed in a future version. Please use standard Open Rosa paths on your server.</string>
```

Comment:
I don't think people know what "OpenRosa" is. Could we say "Please use /formList and /submission on your server"?

Reply:
I thought it looked a little weird but I don't have strong feelings about it. I'll change it.